### DISCUSSION TITLE
http-logger以及tcp-logger插件记录日志信息不够安全

### DISCUSSION THREAD

**Discussion by liangliang4ward** [apache/apisix#5436] [General]
默认的http-logger以及tcp-logger中记录的信息包含了conumser的所有信息，包括授权信息ak,sk。这种方式不太安全，是否能够在记录日志的时候，不记录这些信息。
当然，我知道可以自定义日志格式，但是自定义日志格式还没有默认的信息全。比如没办法配置请求头

**Comment by spacewander** [comment]
We can only log the consumer.name?

**Reply by liangliang4ward** [reply]
> We can only log the consumer.name?

how to do ?

**Reply by spacewander** [reply]
Err, it requires modifying the code.

**Reply by liangliang4ward** [reply]
![image](https://user-images.githubusercontent.com/3895272/140709384-c24fb450-2957-4b1a-a240-da5aef97c7c7.png)

ok, I think can add property to control

**Reply by spacewander** [reply]
Just submit a PR for it: https://github.com/apache/apisix/pull/5445
